bcgov/moh-prime
# PRIME-2012 Fix empty Obo Site form

**How to test:**

1. Go through a normal enrolment, that has a certificate (try for HA Care Setting and non HA Care Setting)
2. In overview go back to certificates page and remove the certificate.
3. Submit and go to overview again (you should have errors to provide either obo site or a certificate)
4. Go to Obo Site page, do nothing and hit back (you should go to certificates page)
5. Provide a certificate and submit to go back to over view (no errors this time)
6. Submit enrolment (You should be able to submit)

**Tested:**
tested the steps above with HA Care Setting and non-HA Care Setting.
